### PR TITLE
Refactored resetUserMfaCmdF to return multierror

### PR DESCRIPTION
**Pull request by AntiGhot**
#### Summary
Returns multi error instead of printing and returning nil

make build  // success
make test   // success
make test-e2e // success

#### Ticket Link

github : https://github.com/mattermost/mattermost-server/issues/21467
JIRA: https://mattermost.atlassian.net/browse/MM-47819

### CODE REVIEW

**Comment by AntiGhot** [comment]
@isacikgoz

File: commands/user.go [hunk 1]
```diff
@@ -616,18 +616,19 @@ func resetUserMfaCmdF(c client.Client, cmd *cobra.Command, args []string) error
...
 	users, err := getUsersFromArgs(c, args)
 	if err != nil {
-		printer.PrintError(err.Error())
+		result = multierror.Append(result, err)
 	}
 
 	for _, user := range users {
 		if _, err := c.UpdateUserMfa(user.Id, "", false); err != nil {
-			printer.PrintError("Unable to reset user '" + user.Id + "' MFA. Error: " + err.Error())
+			result = multierror.Append(result, fmt.Errorf("Unable to reset user '"+user.Id+"' MFA. Error: "+err.Error()))
```
Comment (noxer) [inline commands/user.go:627]: You can use string interpolation here:
```suggestion
			result = multierror.Append(result, fmt.Errorf("Unable to reset user %q MFA. Error: %w", user.Id, err))
```

**Review by noxer — CHANGES_REQUESTED** [review]
Hi @AntiGhot, thank you very much for your contribution. I'd like to request one change. Otherwise the PR looks good.

**Comment by AntiGhot** [comment]
@noxer review comments addressed

**Review by isacikgoz — APPROVED** [review]
Thanks @AntiGhot LGTM 💙

**Review by noxer — APPROVED** [review]
Yes, perfect. Thank you @AntiGhot

**Pull request merged**